crates/dbsp/examples/json.rs

```diff
@@ -0,0 +1,68 @@
+use anyhow::Result;
+use dbsp::{circuit::CircuitConfig, utils::Tup2, Runtime};
+use itertools::Itertools;
+use serde_json::json;
+
+type Json = ijson::IValue;
+
+fn parse_json(val: String) -> Json {
+    serde_json::from_str(&val).expect("failed to deserialize json")
+}
+
+fn json_field(val: Json, field: &str) -> Json {
+    val[field].clone()
+}
+
+fn cast_to_string_json(val: Json) -> String {
+    serde_json::to_string(&val).expect("failed to cast to string")
+}
+
+fn main() -> Result<()> {
```

Comment:
can you sketch the APIs for handling arrays too?

Reply:
can you make this a `#[test]` instead?
Also, can you sketch a more complex app that uses arrays of objects?